examples/90time_conv.py

```diff
@@ -3,19 +3,19 @@
 The programmed SpinCore pulse length does not
 match the actual output pulse length. This example
 finds what pulse length should be fed to SpinCore
-in order to get a pulse length with the actual
-desired pulse length.
+in order to get the desired beta.
 """
 import matplotlib.pyplot as plt
 from numpy import r_
 import pyspecdata as psd
 from SpinCore_pp.pulse_length_conv import prog_plen
 
 # the following choice fails dramatically with the old code
-desired_plen = r_[0:50:50j]
-prog_plen = prog_plen(desired_plen)
+desired_beta = r_[10:150:50j]
+prog_plen = prog_plen(desired_beta)
```

Comment:
This should now be run for two different amplitudes. It's probably easiest to just nest the code in a loop

Reply:
For clarity, delete this line here, and just call the function on the same line where you're plotting (inside the plotting function)